<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主窗体</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../img.qrc">
    <normaloff>:/img/images/user.png</normaloff>:/img/images/user.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="auto_but">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>99</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url()</string>
   </property>
   <property name="text">
    <string>自动测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url()</string>
   </property>
   <property name="text">
    <string>SerialSystem_Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="report_but">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url()</string>
   </property>
   <property name="text">
    <string>测试报告</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manual_but">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>99</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url()</string>
   </property>
   <property name="text">
    <string>手动测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="test">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:4px groove gray;border-radius:10px;padding:10px;background-image:url();</string>
   </property>
   <property name="text">
    <string>测试报告</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>0</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/img/images/serial_logo.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back_but">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>380</y>
     <width>99</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url()</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="main_gif">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>280</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::YearSection</enum>
   </property>
   <property name="displayFormat">
    <string>yyyy年MM月dd日 HH 时 mm 分 ss 秒</string>
   </property>
   <property name="calendarPopup">
    <bool>false</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::LocalTime</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../img.qrc"/>
 </resources>
 <connections/>
</ui>
